packages/build-ui/src/pages/agits/_id/management/artists/models.rs

```diff
@@ -0,0 +1,16 @@
+#[derive(Clone, Debug, PartialEq)]
+
+pub struct Artist {
+    pub id: String,
+    pub artist_name: String,
+    pub artist_mail: String,
```

Comment:
```suggestion
    pub name: String,
    pub email: String,
```

you don't need to explain them for an artist because all field are for an artist